execute_tests.sh

```diff
@@ -14,9 +14,55 @@ SYS_VENV_NAME="${SYS_VENV_NAME:-venv-molecule}"
 SYS_CONSTRAINTS="constraints.txt"
 SYS_REQUIREMENTS="requirements.txt"
 SYS_INVENTORY="${SYS_INVENTORY:-/etc/openstack_deploy/openstack_inventory.json}"
+MOLECULES=()
+
+## Functions -----------------------------------------------------------------
+
+usage() {
+  echo -n "execute_tests [-l] [-m MOLECULE_PATH(S)]
+Execute Molecule tests.
+
+ Options:
+  -l    Enable local testing (Not in CI)
+  -m    Path of single Molecule to execute
+  -h    Display this help and exit
+"
+}
+
+## Parse Args ----------------------------------------------------------------
+
+while getopts ":lm:h" opt;
+do
+  case $opt in
+    l)
+      export MNAIO_SSH="ssh -ttt -oStrictHostKeyChecking=no root@infra1"
```

Comment:
I am not sure I am crazy about this setting. This ties local execution to an MNAIO RPC-O deploy without much explanation. If this setting was used on an AIO environment it would cause the molecules to fail. This will also not be compatible with OSP.

Reply:
@johnduarte Added a new commit to address your feedback. Changed the option name, expanded help documentation and added information to README.md.